config.md

```diff
@@ -20,22 +20,28 @@ The `version` element specifies the version of the OCF specification which the c
 
 Each container has exactly one *root filesystem*, and any number of optional *mounted filesystems*. Both need to be declared in the manifest.
 
-The rootfs string element specifies the path to the root file system for the container, relative to the path where the manifest is. A directory MUST exist at the relative path declared by the field.
+| Element name   | Required | Default | Description
+| :------------: | :------: | :-----: | :---------- 
+| path           |   YES    |   N/A   | The path to the root file system for the container, relative to the path where the manifest is. A directory MUST exist at the relative path declared by the field.
+| readonly       |   NO     | `false` | Indicates wither or not the file system reflected inside the container is read-only or read-write, when `true` processes running inside the container would see the file-system as read-only and not be allowed write access.  Some operating systems may not support read-only root (or OS) file systems and some may not support read-only file systems at all.  
```

Comment:
s/wither/whether/
s/, when `true`/. When `true`,/
I am not sure why you mention "OS" file systems, seems like an unnecessary aside.